### PR TITLE
Getting started - rephrasing several sections

### PR DESCRIPTION
**Pull request by PilarQuesada**
Hi Team,
this PR closes the issue #3265  

## Description
I have modified some text (rephrasing) from the following sections: 
- Use cases
- Log data analysis
- File integrity monitoring
- Rootkits detection
- Active response
- Configuration assessment
- System inventory
- Vulnerability detection
- Cloud security monitoring
- Containers security monitoring
- Regulatory compliance

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [x] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/getting-started/use_cases/active_response.rst [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
+:ref:`Wazuh agent <wazuh_agent>` automates the response to threats by running actions when they are detected. The agent has the ability to block network connections, stop running processes or delete malicious files, among other actions. In addition, it can also run customized scripts developed by the user (for example, Python, Bash or PowerShell).
 
-In order to use this feature, users do define the conditions that will trigger the scripted actions. These conditions usually involve threat detection and assessment. For example, a user could use log analysis rules to detect an intrusion attempt, and an IP reputation database to assess the threat by looking for the source IP address of the attempted connection.
+In order to use this feature, users define the conditions that will trigger the programmed actions, which usually involve the threat detection and assessment. For example, a user could use log analysis rules to detect an intrusion attempt, and an IP reputation database to assess the threat by looking for the source IP address of the attempted connection.
```
Comment (davidfmiranda) [inline source/getting-started/use_cases/active_response.rst:10]:
```suggestion
In order to use this feature, users define the conditions that will trigger the scripted actions, which usually involve threat detection and assessment. For example, a user could use log analysis rules to detect an intrusion attempt, and an IP reputation database to assess the threat by looking for the source IP address of the attempted connection.
```

File: source/getting-started/use_cases/cloud_security.rst [hunk 2]
```diff
@@ -100,20 +100,20 @@ Example of AWS dashboard:
...
 
 - :ref:`Log Analytics API <azure_monitoring_activity>`: The Log Analytics API is a core component of the Azure Monitor service, which is used to aggregate and analyze log data. The sources of such data are cloud applications, operating systems and Azure resources. The Wazuh module for Azure is capable of querying the Log Analytics API, pulling the logs that are collected by the Azure monitor service.
 
-- :ref:`Blob Storage API <azure_monitoring_activity>`: Logs from Azure services can optionally be pushed to Azure Blob Storage. More specifically, you can configure an Azure service to export logs to a container in a storage account created for that purpose. Then, the Wazuh agent can download those logs via its integration with the Blob Storage API.
+- :ref:`Blob Storage API <azure_monitoring_activity>`: Logs from Azure services are optionally pushed to Azure Blob Storage. Specifically, it is possible to configure an Azure service to export logs to a container in a storage account created for that purpose. Afterwards, the Wazuh agent will download those logs via its integration with the Blob Storage API.
```
Comment (davidfmiranda) [inline source/getting-started/use_cases/cloud_security.rst:112]: This is not compulsory, so it should remain as it was previously.

File: source/getting-started/use_cases/cloud_security.rst [hunk 3]
```diff
@@ -158,7 +158,7 @@ More information about how to use Wazuh to monitor Microsoft Azure can be found
 Google Cloud Platform
 ---------------------
 
-Wazuh is capable of monitoring Google Cloud services by pulling events from the Google Pub/Sub messaging service, which is used as a middleware for events ingestion and delivery. This integration helps detect threats targeting your Google Cloud assets.
+Wazuh monitors Google cloud services by pulling events from the Google Pub/Sub messaging service, which is used as a middleware for events ingestion and delivery. This integration helps detect threats targeting your Google Cloud assets.
```
Comment (davidfmiranda) [inline source/getting-started/use_cases/cloud_security.rst:161]:
```suggestion
Wazuh monitors Google Cloud services by pulling events from the Google Pub/Sub messaging service, which is used as a middleware for events ingestion and delivery. This integration helps detect threats targeting your Google Cloud assets.
```

Comment (davidfmiranda) [reply source/getting-started/use_cases/cloud_security.rst:161]: The service is called `Google Cloud` so it must be in capitals

**Review by davidfmiranda — CHANGES_REQUESTED** [review]
Hello @PilarQuesada, I've reviewed your PR and LGTM. Nevertheless, I've made some suggestions.

Regards,

David

File: source/getting-started/components/index.rst [hunk 4]
```diff
@@ -5,22 +5,25 @@
...
 
-In addition to agent-based monitoring capabilities, the Wazuh platform can monitor agent-less devices such as firewalls, switches, routers, network IDS, etc. For example, a system log data can be collected via Syslog, and its configuration can be monitored through periodic probing of its data (e.g. via SSH or through an API).
+In addition to agent-based monitoring capabilities, the Wazuh platform can monitor agent-less devices such as firewalls, switches, routers or network IDS, among others. For example, a system log data can be collected via Syslog, and its configuration can be monitored through periodic probing of its data (e.g. via SSH or through an API).
 
-The diagram below represents the Wazuh components and data flow. It shows the three main components of the solution, which are the :ref:`Wazuh agent <wazuh_agent>`, the :ref:`Wazuh server <wazuh_server>`, and the :ref:`Elastic Stack <elastic_stack>`.
+The diagram below represents the Wazuh components and data flow, showing the three main components of the solution: the :ref:`Wazuh agent <wazuh_agent>`, the :ref:`Wazuh server <wazuh_server>`, and the :ref:`Elastic Stack <elastic_stack>`.
```
[the remark this answers is not in the document]
Comment (PilarQuesada) [reply source/getting-started/components/index.rst:16]: Ok, we'll live the previous version there.

File: source/getting-started/use_cases/cloud_security.rst (hunk 2 above)
Comment (PilarQuesada) [reply source/getting-started/use_cases/cloud_security.rst:112]: The expression has not changed in the new version. It says "it is possible". It's only a rephrasing of the way to express it, but it's not changing the "optional" character of the sentence. It says "are optionally pushed".

File: source/getting-started/components/index.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (PilarQuesada) [reply source/getting-started/components/index.rst]: Done.

**Review by davidfmiranda — APPROVED** [review]
LGTM

**Pull request merged**